include/boost/json/impl/monotonic_resource.ipp

```diff
@@ -79,35 +97,67 @@ monotonic_resource::
 }
 
 monotonic_resource::
-monotonic_resource() noexcept
-    : block_size_(64 * 1024)
+monotonic_resource(
+    std::size_t block_size,
+    std::size_t max_alloc) noexcept
+    : block_size_(block_size),
+      max_alloc_(max_alloc)
 {
 }
 
 monotonic_resource::
 monotonic_resource(
-    std::size_t block_size) noexcept
-    : block_size_(block_size)
+  void* initial_block,
```

Comment:
This parameter has the wrong name, it isn't a "block" in the sense of being prefixed with a `block` in the layout